content/src/thumbnail.js

```diff
@@ -61,12 +78,13 @@ function getImageInfo(canvas) {
 
   // Find the longer edge and make it a square.
   var longerEdge;
+  var diff = Math.abs((imageWidth - imageHeight) / 2);
```

Comment:
Yeah I got what you mean. I was referring to that. 



Best regards, Liu Xinan

On Fri, Jul 17, 2015 at 10:03 AM, David Bau <notifications@github.com>
wrote:

>> @@ -61,12 +78,13 @@ function getImageInfo(canvas) {
>>  
>>    // Find the longer edge and make it a square.
>>    var longerEdge;
>> +  var diff = Math.abs((imageWidth - imageHeight) / 2);
> It is OK not to do anything special case when there is just an unmoved turtle: the icon can be just a turtle.
> I am asking about the situation where even the turtle is not present and the screen is totally blank: it seems that a totally blank icon (and maybe even an "almost blank" icon) would be confusing to see and use within a grid of icons.
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/PencilCode/pencilcode/pull/155/files#r34857025